docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,355 @@
+
+##### Issue: Event Not Triggering on User Action
+
+If an event is not firing, verify that the desired action (e.g., 'click', 'change') is configured for the specific event (e.g., `PRODUCT_CLICK`).
+
+1. Check the Configuration
+
+Spryker provides default configurations for built-in components. For new or modified components, you need to add the appropriate event configuration.
+
+```twig
+{% raw %}{% block eventTracker %}{% endraw %}
+    {% raw %}{% set events = {{% endraw %}
+        list: events.list | merge([
+            {
+                event: 'NEEDED_EVENT_NAME', // e.g., PRODUCT_CLICK
+                name: 'NEEDED_EVENT_LISTENER', // e.g., click/change
+                triggers: [...event triggers data],
+            },
+        ]),
+        data: events.data,
+    } %}
+
+    {{ parent() }}
+{% raw %}{% endblock %}{% endraw %}
+```
+
+Refer to the [API documentation](https://github.com/spryker-shop/traceable-event-widget/blob/master/src/SprykerShop/Yves/TraceableEventWidget/Theme/default/components/molecules/traceable-events-orchestrator/README.md) for more details.
+
+2. Check Event Selector
+
+Spryker includes default CSS selectors. If selectors have changed, update the configuration accordingly.
+
+```twig
+{% raw %}{% block eventTracker %}{% endraw %}
+    {% raw %}{% set events = {{% endraw %}
+        list: events.list | merge([
+            {
+                event: 'NEEDED_EVENT_NAME', // e.g., PRODUCT_CLICK
+                name: 'NEEDED_EVENT_LISTENER', // e.g., click/change/load
+                triggers: [
+                    {
+                        selector: 'new_css_selector_path', // The element selector to monitor
+                        /* event data configuration */
+                    },
+                ],
+            },
+        ]),
+        data: events.data,
+    } %}
+
+    {{ parent() }}
+{% raw %}{% endblock %}{% endraw %}
+
+```
+
+##### Issue: Incorrect Event Payload
+
+You can view the event payload in the console under `Adapter Data:`. If the payload is incorrect, check the static and dynamic data configurations.
+
+1. Static Data
+
+Adjust static data in the `eventTracker` block as needed:
+
+```twig
+{% raw %}{% block eventTracker %}{% endraw %}
+    {% raw %}{% set events = {{% endraw %}
+        list: events.list,
+        data: events.data | merge({
+          existing_key_to_override: New Data,
+          new_key: New Data,
+        }),
+    } %}
+
+    {{ parent() }}
+{% raw %}{% endblock %}{% endraw %}
+```
+
+2. Dynamic Data
+
+For adding dynamic data, refer to the [API documentation](https://github.com/spryker-shop/traceable-event-widget/src/SprykerShop/Yves/TraceableEventWidget/Theme/default/components/molecules/traceable-events-orchestrator/README.md). Adjust the configuration as needed for specific triggers.
+
+```twig
```

Comment:
Add file path

Reply:
fixed the link for "API documentation". Did you mean this?

Reply:
I'm referring to this twig file. Is this any particular file or just an example?